snippets/terraform-modification-disclaimer.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/terraform/rosa-sts-creating-a-cluster-quickly-terraform.adoc
+// * rosa_planning/rosa-understanding-terraform.adoc
+
+:_mod-docs-content-type: SNIPPET
+[IMPORTANT]
+====
+Use caution when modifying Terraform created resources. For more information, see link:https://docs.openshift.com/rosa/rosa_planning/rosa-understanding-terraform.html[Considerations when using Terraform]
```

Comment:
As a reminder/heads up, make sure to direct the link to the specific topic once you are ready to publish (ie "Considerations when using Terraform").